spec/rails_helper.rb

```diff
@@ -33,6 +34,8 @@
   exit 1
 end
 RSpec.configure do |config|
+  config.include AuthHelpers::Includables, type: :request
+  config.extend AuthHelpers::Extensions, type: :request
```

Comment:
Common practice is to move such extensions directly to the support file. Like:

```
# support/auth_helpers.rb
module AuthHelpers
  # ...
end

RSpec.configure do |config|
  config.include ...
end
```

Reply:
thanks, I have refactored that part.